bindings/python/visualize/meshcat_visualizer.py

```diff
@@ -636,8 +636,8 @@ def load_model(self, model: idyn.Model, base_frame = None, model_name="iDynTree"
         if base_frame is None:
             self.model[model_name].computeFullTreeTraversal(self.traversal[model_name])
         else:
-            base_frame_index = self.model[model_name].getFrameIndex(base_frame)
-            self.model[model_name].computeFullTreeTraversal(self.traversal[model_name], base_frame_index)
+            base_link_index = self.model[model_name].getFrameIndex(base_link)
```

Comment:
If we are passing a link, we should use getLinkIndex:

```suggestion
            base_link_index = self.model[model_name].getLinkIndex(base_link)
```

the result is exactly the same if the passed string is a link, but you get a LINK_INVALID_INDEX if the string does not correspond to a link but just an additional frame.